diff --git a/libintervalxt/configure.ac b/libintervalxt/configure.ac
index fe61b3e6..01023384 100644
--- a/libintervalxt/configure.ac
+++ b/libintervalxt/configure.ac
@@ -56,7 +56,9 @@ AX_CXX_CHECK_LIB([fmt], [fmt::v6::getpagesize()], , [
     AX_CXX_CHECK_LIB([fmt], [fmt::v8::report_system_error(int code=0, const char* message=nullptr)], , [
       AX_CXX_CHECK_LIB([fmt], [fmt::v9::report_system_error(int code=0, const char* message=nullptr)], , [
         AX_CXX_CHECK_LIB([fmt], [fmt::v10::report_system_error(int code=0, const char* message=nullptr)], , [
-          AX_CXX_CHECK_LIB([fmt], [fmt::v11::report_system_error(int code=0, const char* message=nullptr)], , [AC_MSG_ERROR([fmt library not found])], [-lfmt])
+          AX_CXX_CHECK_LIB([fmt], [fmt::v11::report_system_error(int code=0, const char* message=nullptr)], , [
+            AX_CXX_CHECK_LIB([fmt], [fmt::v12::report_system_error(int code=0, const char* message=nullptr)], , [AC_MSG_ERROR([fmt library not found])], [-lfmt])
+          ], [-lfmt])
         ], [-lfmt])
       ], [-lfmt])
     ], [-lfmt])
